### ISSUE TITLE
[wavedeform] docs need updating (Trac #1193)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1187]
I can tell the docs need updating because they mention DOMSimulator hacks, which are not in the code anymore.

Migrated from https://code.icecube.wisc.edu/ticket/1193
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "I can tell the docs need updating because they mention DOMSimulator hacks, which are not in the code anymore.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[wavedeform] docs need updating", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-08-19T16:59:56", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T14:09:30: 
* jbraun changed status from "new" to "closed"
* jbraun changed resolution from "" to "fixed"
* jbraun commented:

> In changeset 136615:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136615"
> The DOMSimulator parameter was the only part of the docs that was out of date.  Closes #1187
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
